CMakeLists.txt

```diff
@@ -224,7 +224,7 @@ if(ENABLE_QT)
     endif()
 
     foreach(M ${QT5_REQUIRED_MODULES})
-        find_package(Qt5${M} ${QT5_MINIMUM} QUIET)
+        find_package(Qt5${M} ${QT5_MINIMUM} REQUIRED)
         if(Qt5${M}_FOUND)
```

Comment:
This check doesn't make sense since the above command will fail configuration if package is not found and we'll never reach it.